docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -113,15 +126,25 @@ After preparing either the libvirt or docker prerequisites using the instruction
       should be compatible.
 
 #. Next, cd into the ``devel`` directory. The Pulp project provides an example Vagrantfile that you can
-   use as a starting point by copying it. After you've done that, you can begin provisioning your
+   use as a starting point by copying it.::
```

Comment:
When rendered this has a dangling colon after it. If you use just 1 colon it still shows and the section below doesn't escape. (I just tried that). I'm not sure how to force the below part here without the double colon. One option to fix this is to rephrase the sentence to have the colon make sense (with no period).